### PR TITLE
Updated the instructions to build docs locally

### PR DESCRIPTION
**Pull request by javiereguiluz**
This finishes #6395.

### CODE REVIEW

File: contributing/documentation/overview.rst [hunk 1]
```diff
@@ -258,6 +258,24 @@ link displayed for Platform.sh service.
...
+**Step 1.** Install `pip`_ as explained in the `pip installation`_ article.
+
+**Step 2.** Install `Sphinx`_ and `Sphinx Extensions for PHP and Symfony`_
+(depending on your system, you may need to execute this command as root user):
+
+.. code-block:: bash
+
+    $ pip install sphinx~=1.3.0 git+https://github.com/fabpot/sphinx-php.git
+
+**Step 3.** Run ``make html`` and view the generated documentation in the
```
Comment (snoek09) [inline contributing/documentation/overview.rst]: Maybe it should be mentioned that `make html` has to be run inside the `_build' dir.

**Comment by snoek09** [comment]
:+1: 

Status: Reviewed

File: contributing/documentation/overview.rst (hunk 1 above)
Comment (javiereguiluz) [inline contributing/documentation/overview.rst]: Nice catch! Thank you. I've reworded this a bit in a7b4d32.

File: contributing/documentation/overview.rst [hunk 2]
```diff
@@ -258,6 +258,30 @@ link displayed for Platform.sh service.
...
+**Step 1.** Install `pip`_ as explained in the `pip installation`_ article.
+
+**Step 2.** Install `Sphinx`_ and `Sphinx Extensions for PHP and Symfony`_
+(depending on your system, you may need to execute this command as root user):
+
+.. code-block:: bash
+
+    $ pip install sphinx~=1.3.0 git+https://github.com/fabpot/sphinx-php.git
+
+**Step 3.** Run the following command to build the documentation in HTML format:
```
Comment (wouterj) [inline contributing/documentation/overview.rst]: please use an ordered list here

File: contributing/documentation/overview.rst [hunk 3]
```diff
@@ -258,6 +258,30 @@ link displayed for Platform.sh service.
     Only Pull Requests to maintained branches are automatically built by
     Platform.sh. Check the `roadmap`_ for maintained branches.
 
+Build Documentation Locally
```
Comment (xabbuh) [inline contributing/documentation/overview.rst]: Build the Documentation Locally

**Comment by yceruto** [comment]
I've tried this in Windows and not work for me :/

```
...\symfony-docs\_build>make html
Running Sphinx v1.3.1
making output directory...
A Translator for the html builder is changed.
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
WARNING: master file ...\symfony-docs\_build\index.rst not found
looking for now-outdated files... none found
no targets are out of date.
build succeeded, 1 warning.

Build finished. The HTML pages are in _build/html.
```
... and the path `_build/html` is empty.

:question:

**Comment by javiereguiluz** [comment]
@yceruto the error seems to be in this line:

```
WARNING: master file ...\symfony-docs\_build\index.rst not found
```

**Comment by wouterj** [comment]
`make html` should be run in `\symfony-docs`. (using `"_build/mak" html` or `_build\make html`)

File: contributing/documentation/overview.rst [hunk 4]
```diff
@@ -258,6 +258,29 @@ link displayed for Platform.sh service.
...
+
+   .. code-block:: bash
+
+        $ pip install sphinx~=1.3.0 git+https://github.com/fabpot/sphinx-php.git
+
+#. Run the following command to build the documentation in HTML format:
+
+   .. code-block:: bash
+
+       $ _build\make html
```
Comment (wouterj) [inline contributing/documentation/overview.rst]: on Unix, it should be `./_build/make html` IIRC

**Comment by yceruto** [comment]
@wouterj `symfony-docs>_build\make html` work perfect, thanks.

On the other hand, inside `symfony-docs/_build/_theme/_templates/layout.html` was declared the main assets to external urls:
* css `http://symfony.com/css/compiled/v5/all.css?v=4`
* js `http://symfony.com/js/v5/all.js?v=4`

this not work offline, any solution ?

**Comment by yceruto** [comment]
The same for all *.svg paths inside `http://symfony.com/css/compiled/v5/all.css`, they aren't offline.

**Comment by yceruto** [comment]
I just created a separate issue for these questions, because they are out of scope of this PR. https://github.com/symfony/symfony-docs/issues/6844

File: contributing/documentation/overview.rst [hunk 5]
```diff
@@ -258,6 +258,29 @@ link displayed for Platform.sh service.
     Only Pull Requests to maintained branches are automatically built by
     Platform.sh. Check the `roadmap`_ for maintained branches.
 
+Build the Documentation Locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively you can build the documentation in your own computer following
+these steps:
```
Comment (yceruto) [inline contributing/documentation/overview.rst]: To be honest, I was confused in thinking that could build the documentation locally and use it completely "offline", which it's not possible right now and probably never will https://github.com/symfony/symfony-docs/issues/6844#issuecomment-237632673. Maybe we can add the subject "only for testing purpose" somehow? thanks

File: contributing/documentation/overview.rst (hunk 4 above)
Comment (yceruto) [inline contributing/documentation/overview.rst]: Maybe this part should be split in two sections ? Unix and Windows ?

**Comment by wouterj** [comment]
👍

File: contributing/documentation/overview.rst [hunk 6]
```diff
@@ -258,6 +258,33 @@ link displayed for Platform.sh service.
     Only Pull Requests to maintained branches are automatically built by
     Platform.sh. Check the `roadmap`_ for maintained branches.
 
+Build the Documentation Locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Alternatively you can build the documentation in your own computer for testing
```
Comment (xabbuh) [inline contributing/documentation/overview.rst]: on your own computer

**Comment by weaverryan** [comment]
Thanks Javier!

**Pull request merged**